exp3/run_recognition.py

```
@@ -123,25 +123,27 @@ def main():
-        digit_correct = confusion_matrix[digit][digit]
-        digit_accuracy = (digit_correct / digit_total * 100) if digit_total > 0 else 0
-        print(f"  数字 {digit}: {digit_correct}/{digit_total} ({digit_accuracy:.2f}%)")
+    print(f"\n各标签识别准确率:")
+    for label in config.LABELS:
+        label_total = len(test_files[label])
+        label_correct = confusion_matrix[label][label]
+        label_accuracy = (label_correct / label_total * 100) if label_total > 0 else 0
+        print(f"  标签 {label}: {label_correct}/{label_total} ({label_accuracy:.2f}%)")
     
     # 打印混淆矩阵
     print(f"\n混淆矩阵 (行:真实, 列:预测):")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)